tools/bump_ocp_releases.py

```diff
@@ -33,6 +33,10 @@
     r"curl https://api.openshift.com/api/upgrades_info/v1/graph\?channel\=candidate-{version}\&arch\={architecture}"
     " | jq '[.nodes[]]'"
 )
+OCP_INFO_MULTIARCH_CALL = (
+    r"curl https://multi.ocp.releases.ci.openshift.org/graph"
```

Comment:
let's transform this to usage of ``requests``, I think it's the more Pythonic way rather than transferring the job to some OS command

Reply:
same goes with the ``jq`` part, ``response.json()`` can help you with getting the relevant dictionary with all the data